jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py

```diff
@@ -279,9 +279,11 @@ def _on_flux_unit_changed(self, msg):
 
         if self.spectrum_viewer.state.y_display_unit != yunit:
             self.spectrum_viewer.state.y_display_unit = yunit
-            self.hub.broadcast(GlobalDisplayUnitChanged('flux',
-                                                        flux_or_sb,
-                                                        sender=self))
+            self.hub.broadcast(
+                GlobalDisplayUnitChanged(
+                    "flux" if name == "flux_unit_selected" else "sb", flux_or_sb, sender=self
+                    )
+                )
```

Comment:
@gibsongreen - does this affect you in #3111?  Will we need to emit messages for (flux AND surface brightness) OR (solid angle AND surface brightness)?

Reply:
I believe it will be solid_angle and flux, but in my PR solid_angle isn't computed/changed in this function currently (since it is either /sr if surface brightness, or, / pix is the cube loaded is flux (which really should spaxel). Right now it is computed here:
https://github.com/spacetelescope/jdaviz/blob/196659bdff7eb2f409000d2cc1c342ebe621bcaa/jdaviz/configs/specviz/plugins/unit_conversion/unit_conversion.py#L181

When enabling changes to the solid_angle unit then there would be addition of something like 'solid_angle_unit_selected' that would need to be observed in this function